tests/cupyx_tests/scipy_tests/fft_tests/test_fft.py

```diff
@@ -624,17 +876,28 @@ def test_irfft_plan(self, dtype):
             _fft_module(cp).irfft(x, n=self.n, axis=self.axis,
                                   norm=self.norm, plan='abc')
 
+    @unittest.skipIf(scipy_fft is None, 'need scipy >= 1.4.0')
+    @testing.for_all_dtypes(no_complex=True)
+    @testing.numpy_cupy_allclose(rtol=1e-4, atol=1e-6, accept_error=ValueError,
+                                 contiguous_check=False)
+    def test_irfft_backend(self, xp, dtype):
+        x = testing.shaped_random(self.shape, xp, dtype)
+        x_orig = x.copy()
+        backend = 'scipy' if xp is np else cp_fft
+        with scipy_fft.set_backend(backend):
+            out = scipy_fft.irfft(x, n=self.n, axis=self.axis, norm=self.norm)
+        testing.assert_array_equal(x, x_orig)
+        return _correct_np_dtype(xp, dtype, out)
+
 
 @testing.parameterize(
     {'shape': (3, 4), 's': None, 'axes': None, 'norm': None},
-    {'shape': (3, 4), 's': (1, None), 'axes': None, 'norm': None},
```

Comment:
Tests with `s=(..., None)` for multi-dimensional `rfft` are removed, because pocketfft raises an error (`shape must be a scalar or iterable of integers`).